site/docs/v1/tech/apis/identity-providers/links.adoc

```diff
@@ -23,7 +23,10 @@ This page contains the APIs that are used to manage Links that establish a relat
 This API is used to create a link between a FusionAuth user and a user in a 3rd party identity provider. This API may be useful when you already know the unique Id of a user in a 3rd party identity provider and the corresponding FusionAuth user.
 
 === Request
-
+[NOTE.since]
+====
+This API endpoint has been available since 1.43.0
```

Comment:
```suggestion
This API endpoint has been available since 1.43.0.
```

Reply:
Weird to have this two places. I think it would make sense to leave this here and leave line 11 at 1.28.